change/@fluentui-react-drawer-3113d74d-da7e-4856-a19e-42fb5793fa81.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29360&commitId=750251cd8d1558c63a623559c035a190f98b5f59&env=prod&clientType=FLUENTUI)


  <details><summary><b>Menu Converged - long content</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu Converged - long content.default - RTL.chromium.png|245| Changed|
|Menu Converged - long content.default.chromium.png|245| Changed|

</details>


  <details><summary><b>Menu Converged - split item</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu Converged - split item.default.chromium.png|68| Changed|
|Menu Converged - split item.default.submenu open.chromium.png|67| Changed|
|Menu Converged - split item.default - RTL.submenu open.chromium.png|67| Changed|
|Menu Converged - split item.default - RTL.chromium.png|68| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>